runtime/common/src/lib.rs

```diff
@@ -483,3 +483,56 @@ pub mod xcm_transactor {
 		}
 	}
 }
+
+pub mod foreign_investments {
+	use cfg_primitives::Balance;
+	use cfg_traits::SimpleCurrencyConversion;
+	use cfg_types::{
+		fixed_point::{FixedPointNumberExtension, Rate},
+		tokens::CurrencyId,
+	};
+	use frame_support::pallet_prelude::PhantomData;
+	use sp_runtime::{traits::Get, DispatchError};
+
+	/// Simple stable coin amount converter from one stable to another. Provides
+	/// a synchronous estimation of the amount of one stable currency in
+	/// another, e.g., the worth of 100 EthWrappedDai in USDC.
+	///
+	/// For now, converts any `ForeignAsset` into another with the configured
+	/// rate.
+	///
+	/// NOTE: Should be deprecated ASAP!
+	// TODO(@review): Can we determine whether a ForeignAsset is a stable coin at
+	// this point of time?
```

Comment:
I also wondered about that, as we need it also for the `OrderBook`. I would like to have it as part of the registry TBH. What do the others think?

We would also need an identifier to which off-chain value the coin is pegged to.